benchmarks/test_haloprofile.py

```diff
@@ -146,6 +146,8 @@ def test_weak_lensing_functions():
         np.log(rmax/rmin) * np.arange(data.shape[0]) / (data.shape[0]-1))
 
     r_al = r / a_lens
+    len_r = len(r)
+    a_source = [a_source]*len_r
```

Comment:
I don't understand this. Why do you need `a_source` to be of the same shape as `r`?

Reply:
The benchmark data `data` has the same length as `r` (500 values), so requiring`a_source` to have the same length as `r` ensures the outputs from the calculations below have the correct dimension. Otherwise, there will be an error on numpy broadcasting.